lib/project_and_version_value.rb

```diff
@@ -74,8 +74,8 @@ def version_costs(proj_id, version_id)
   # @param [date] basis_date basis date
   # @return [Issue] issue object
   def incomplete_project_issues(proj, basis_date)
-    Issue.cross_project_scope(proj, 'descendants')
-      .where('start_date IS NOT NULL AND start_date <= ? AND due_date IS NOT NULL AND (closed_on IS NULL OR closed_on > ?)', basis_date, basis_date.end_of_day)
+    Issue.cross_project_scope(proj, "descendants")
+      .where("start_date IS NOT NULL AND start_date <= ? AND due_date IS NOT NULL AND (closed_on IS NULL OR closed_on > ?)", basis_date, basis_date.end_of_day)
```

Comment:
Line is too long. [159/80]<br>Place the . on the previous line, together with the method call receiver.